common/api/core-backend.api.md

```diff
@@ -2188,69 +2186,6 @@ export class GroupModel extends GroupInformationModel {
     static insert(iModelDb: IModelDb, parentSubjectId: Id64String, name: string): Id64String;
 }
 
-// @internal
-export class HubMock {
```

Comment:
It looks like you merged master in and reran `extract-api` without first running `build`. Please build then rerun extract-api.